lib/widget/heist_end.dart

```diff
@@ -59,21 +60,21 @@ class HeistEndState extends State<HeistEnd> {
                   style: buttonTextStyle,
                 ),
                 onPressed:
-                    completingHeist ? null : () => _store.dispatch(new CompleteHeistAction()),
+                    completingHeist ? null : () => _store.dispatch(new CompleteHauntAction()),
```

Comment:
completingHaunt

The previous method should be _hauntDecisions and there are a few other methods starting with _heist